README.md

```diff
@@ -220,6 +220,10 @@ A: If you've followed the [setup instructions](https://docs.solanalabs.com/opera
 - [solana_validator_set_identity_scripts](https://github.com/pzupan/solana_validator_set_identity_scripts)
 - [Guide: How to Switch from Agave/Jito to Frankendancer (Staking Facilities)](https://stakingfacilities.com/blog/guide-how-to-switch-from-agave-jito-to-frankendancer)
 
+## Validator Hotswap & Failover
+- [SVS - Switch, Alert and Autofailover by huisky](https://github.com/huiskylabs/solana-validator-switch)
+- [Pumpkin's Pool Identity Switch Guide](https://pumpkins-pool.gitbook.io/pumpkins-pool)
+
```

Comment:
I think this is in the default anza docs now so let’s link that and maybe also link keyless operation?